src/components/ReviewContainer.jsx

```diff
@@ -0,0 +1,33 @@
+import { useState, useEffect } from "react"
+import {fetchReviews} from "../api"
+import { ReviewCard } from "./ReviewCard"
+import "../styles/ReviewContainer.css"
+
+export const ReviewContainer = () => {
+
+    const [reviews, setReviews] = useState([])
+
+    useEffect(() => {
+        fetchReviews()
+        .then((returnedReviews) => {
+            setReviews(returnedReviews)
+        })
+    }, [])
+
+    return(
```

Comment:
Use conditional rendering here to let the user know if reviews are still loading.